gui/tools/autorepl.py

```diff
@@ -314,12 +314,13 @@ def write_results():
     sshproc = pipeopen('%s %s' % (sshcmd, rzfscmd))
     output, error = sshproc.communicate()
     remote_zfslist = {}
-    for i in re.sub(r'[ \t]+', ' ', output, flags=re.M).splitlines():
-        data = i.split()
+    for i in re.sub(r'\t+', ' ', output, flags=re.M).splitlines():
```

Comment:
```
output = 'beverages/vodka smirnoff/0.5l\toff\n'

for i in re.sub(r'\t+', ' ', output, flags=re.M).splitlines():
    readonly_value = i.split()[-1]
    data = [i.split(readonly_value)[0].rstrip(), readonly_value]
    print(data)

['beverages/vodka smirn', 'off']
# But I expected it to be ['beverages/vodka smirnoff/0.5l', 'off']
```

Reply:
This works as expected for me.

Apr 22 08:33:04 truenas /autorepl.py: [tools.autorepl:318] data = ['receiving', 'on']
Apr 22 08:33:04 truenas /autorepl.py: [tools.autorepl:318] data = ['receiving/DatasetWithUpperCase', 'on']
Apr 22 08:33:04 truenas /autorepl.py: [tools.autorepl:318] data = ['receiving/dataset_with_underscores', 'on']
Apr 22 08:33:04 truenas /autorepl.py: [tools.autorepl:318] data = ['receiving/more spaces', 'on']
Apr 22 08:33:04 truenas /autorepl.py: [tools.autorepl:318] data = ['receiving/more spaces/0.5l', 'on']

Reply:
@yocalebo try dataset that contains `on` in the middle of it's name for readonly=on dataset or `off` in the middle of it's name for readonly=off

Reply:
@themylogin sorry for delay. Please let me know if final commit makes sense.